deploy/grafana/grafana-with-grafana-operator/install-grafana-with-operator.sh

```diff
@@ -57,21 +63,68 @@ kubectl apply --namespace="${GRAFANA_NAMESPACE}" -f <( \
     envsubst \
 )
 
-echo "Waiting to start"
-sleep 60
+echo "Waiting to start Grafana custom resource $GRAFANA_NAME"
+sleep 10
```

Comment:
Timeouts are system specific. E.g. if image should be downloaded, then it may be more than 10 seconds.
Please add a true wait loop checking for certain resource to be properly processed